test/periphery/integration/merkle-campaign/shared/claim/claim.tree

```diff
@@ -15,6 +15,5 @@ claim.t.sol
       │  └── when the Merkle proof is not valid
       │     └── it should revert
       └── given the claim is included in the Merkle tree
-         ├── it should mark the index as claimed
-         ├── it should create a stream
-         └── it should emit a {Claim} event
+         └── it should mark the index as claimed
```

Comment:
And then reverting these changes if we use the above suggestion. Reason is that it will be tested via override function.

Reply:
this one is not true for each campaign `├── it should create a stream`

Reply:
You're right. I should have said that add the following branches here:

```tree
- it should mark the index as Claimed
- It should emit claim event 
- It should emit ERC-20 transfer event
```

Even though all these will be verified in the Child contracts' implementation, we could it add it here

Reply:
Added my suggestion in https://github.com/sablier-labs/v2-core/pull/1016.